### PR TITLE
Pull request (Maxim Diachok) for exams

### PR DESCRIPTION
**Pull request by BbIXOD**
1. count.js
2. replace.js

### CODE REVIEW

**Review by kol-oss — APPROVED** [review]
LGTM. Nothing to add.

File: Tasks/count.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-  sum = 0;
-  {
-  }
-  keys = Object.keys(obj);
-  keys.forEach((key) => {
-    [];
-    const value = obj[key];
+const count = obj => {
+  let sum = 0;
+  const values = Object.values(obj);
```
Comment (tshemsedinov) [inline Tasks/count.js:6]: Alternatively you can use `reduce` to sum values

File: Tasks/replace.js [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
-        src = src.substring(_index + substr.length, src.length);
-        res += start + newstr;
-      }
-    } while (true);
   }
+
+  let source = str;
+  let res = '';
+
+  while (true) {
```
Comment (tshemsedinov) [inline Tasks/replace.js:12]: It's even `replaceAll` :smiley:

Comment (BbIXOD) [reply Tasks/replace.js:12]: I know but we don't have to change realization. Also function, that contains only one line it is weird. I thought that here another realization of replaceAll

**Pull request closed without merging**